### DISCUSSION TITLE
network changd error

### DISCUSSION THREAD

**Discussion by sriharsh1104** [WalletConnect/walletconnect-monorepo#4187] [Developer Support]
![Screenshot from 2024-01-30 11-17-28](https://github.com/WalletConnect/walletconnect-monorepo/assets/95360816/da41cd22-a5de-4b16-80b7-3efc48843290)
in this above screenshot i have setup switch network for binace testnet only but it is working fine but giving a uncaught error network changed 56 to mainnet which i can't able to unterstand how to resolve this . if any solution plz guide with implemention with reason behind it .

**Comment by sriharsh1104** [comment]
i am using library walllet /ethers for web3 modal

**Comment by glitch-txs** [comment]
please provide a minimal reproducible example of the issue
